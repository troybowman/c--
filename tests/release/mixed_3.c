../decl/mixed_3.c