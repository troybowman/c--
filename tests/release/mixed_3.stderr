error: function symbol redeclared at line 3 (previous declaration at line 1)
error: variable x redeclared at line 5 (previous declaration at line 3)
syntax error, line: 9
